vendor/templates/vendor/manage/dashboard.html

```diff
@@ -111,13 +111,13 @@ <h3>{% trans 'Most Recent Sales' %}</h3>
 
           <tbody>
             {% for item in object_list %}
-            <tr>
+            <tr>              
               <td><a class="text-primary" href="{% url 'vendor_admin:manager-order-detail' item.uuid %}">{{ item.pk }}</a></td>
-              <td>{{ item }}</td>
+              <td>{{ item.get_invoice_display }}</td>
               <td>{{ item.get_status_display }}</td>
               <td>{{ item.created }}</td>
               <td>{{ item.get_currency_display }}</td>
-              <td>{{ item.total|default:"0.00" }}</td>
+              <td>$ {{ item.total|floatformat:2|default:"0.00" }}</td>
```

Comment:
Total should handle its own default since it is currency dependent.  It should default to 0.0 if there is nothing in the invoice or if everything in it is free.  Should also look into a currency localization filter.

Default values should rarely be handled in template.

Reply:
removed default filter and added get_total_display